### ISSUE TITLE
Request for a new bugfix release

### ISSUE THREAD

**Issue by ghisvail** [spyder-ide/qtpy#78]
Please consider making a new (bugfix) release (soon) which include commit https://github.com/spyder-ide/qtpy/commit/088c58e7d03e81911735d72bb81d6538cf87f856.

Right now, this bug [forces](https://github.com/pyzo/pyzo/issues/440#issuecomment-250397488) the `pyzo` project to vendor `qtpy` instead of depending on it. A new stable release would allow us to place a versioned dependency on `qtpy` and ultimately drop the vendoring.

Thanks.

**Comment by ccordoba12** [comment]
Ok, I'll do it next week.

**Comment by MatthieuDartiailh** [comment]
Is this still planned ?

**Comment by ccordoba12** [comment]
Yep, I'm working on it right now. Sorry for the delay in doing it :-)

**Comment by ghisvail** [comment]
Great, I'll await the new release on PyPI.
